operator/Makefile

```diff
@@ -441,6 +441,7 @@ bundle-post-process: test-bundle-helpers operator-sdk ## Post-process CSV file t
 	mkdir -p build/
 	rm -rf build/bundle
 	cp -a bundle build/
+	cp ../config-controller/config/crd/bases/*.yaml build/bundle/manifests/
```

Comment:
If possible, please list files by name, rather than use `*` here. It will make it easier to look for references and spot issues if ($DEITY forbid) there ever are clashes. Perhaps use `cp < /dev/null -iv` as well?

Reply:
I'm not sure `-i` is helpful, but happy to discuss in a follow up.

Reply:
I hoped `-i` would prevent overwriting a file present in `bundle/manifests/` in case another file with the same name ever appeared in `../config-controller/config/crd/bases` @kylape .